packages/connect-examples/webextension/trezor-usb-permissions.html

```diff
@@ -32,6 +32,9 @@
         </style>
     </head>
     <body>
-        <script type="text/javascript" src="vendor/trezor-usb-permissions.js"></script>
+        <script
+            type="text/javascript"
+            src="node_modules/@trezor/connect-web/lib/webextension/trezor-usb-permissions.js"
```

Comment:
is it working in browser?

Reply:
just tried again on my machine and it worked.